src/stores/store/eth1/local/local.go

```diff
@@ -406,6 +412,13 @@ func getEIP712EncodedData(typedData *core.TypedData) (string, error) {
 	return fmt.Sprintf("\x19\x01%s%s", domainSeparatorHash, typedDataHash), nil
 }
 
+// getEIP191EncodedData encodes the given message that can be later recovered
+// with the given validator. following EIP-171 spec version 0x0
+func getEIP191EncodedData(address string, msg []byte) string {
+	return fmt.Sprintf("\x19\x00%s%s", address, string(msg))
```

Comment:
Why did you choose to concatenate data to sign that way?
```
Version 0 has <20 byte address> for the version specific data, and the address is the intended validator. In the case of a Multisig wallet, that is the wallet's own address .
```

instead of following this one?
```
Additionally, 0x19 has been chosen because since ethereum/go-ethereum#2940 , the following is prepended before hashing in personal_sign:
```

In addition I would rather work with byes all along to avoid enconding issues

Reply:
I did choose to keep that format to be consistent with the already existing getEIP712EncodedData. But I agree with you bytes would be better. Should we change getEIP712EncodedData too then ?